ts/sagas/messages.ts

```diff
@@ -0,0 +1,126 @@
+import { BasicResponseType } from "italia-ts-commons/lib/requests";
+import { all, call, Effect, put, select, takeLatest } from "redux-saga/effects";
+
+import { Messages } from "../../definitions/backend/Messages";
+import { MessageWithContent } from "../../definitions/backend/MessageWithContent";
+import { MessageWithoutContent } from "../../definitions/backend/MessageWithoutContent";
+import { ServicePublic } from "../../definitions/backend/ServicePublic";
+import { BackendClient } from "../api/backend";
+import { apiUrlPrefix } from "../config";
+import { MESSAGES_LOAD_REQUEST } from "../store/actions/constants";
+import {
+  loadMessagesFailure,
+  loadMessagesSuccess
+} from "../store/actions/messages";
+import { sessionTokenSelector } from "../store/reducers/session";
+
+export interface MessagesListObject {
+  [key: string]: MessageWithoutContent;
+}
+
+export type MessagesIdsArray = ReadonlyArray<string>;
+
+export interface NormalizedMessagesResponse {
+  messages: MessagesListObject;
+  ids: MessagesIdsArray;
+}
+
+export const INITIAL_NORMALIZED_MESSAGES_RESPONSE: NormalizedMessagesResponse = {
+  messages: {},
+  ids: []
+};
+
+function createNormalizedMessagesResponse(
+  messages: ReadonlyArray<MessageWithoutContent>
+): NormalizedMessagesResponse {
+  return messages.reduce((result, value) => {
+    return {
+      messages: {
+        ...result.messages,
+        [value.id]: {
+          ...value
+        }
+      },
+      ids: [...result.ids, value.id]
+    };
+  }, INITIAL_NORMALIZED_MESSAGES_RESPONSE);
+}
+
+function* loadMessage(sessionToken: string, id: string): Iterator<Effect> {
```

Comment:
I would pass an instance of `backendClient` to this method, making it possible to reuse one instance in multiple operations (also making the function pure and testing much easier)